BazelExtensions/extensions.bzl

```diff
@@ -5,6 +5,18 @@
 # Acknowledgements
 
 AcknowledgementProvider = provider()
+_EmptyObjcProvider = apple_common.new_objc_provider()
+
+def _make_objc_interface_provider(header, module_map=None):
+    if hasattr(_EmptyObjcProvider, "header"):
+        return apple_common.new_objc_provider(
+            header=header,
```

Comment:
Some of this has inconsistent formatting - the CI should run buildifier to improve/guard against this.